piksi_tools/console/sbp_relay_view.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# Copyright (C) 2014 Swift Navigation Inc.
+# Contact: Colin Beighley <colin@swift-nav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+from traits.api import HasTraits, String, Button, Instance, Int, Bool, \
```

Comment:
@mookerji : possibly but thats not the Traits convention, better to be consistent with the rest of the code and typical library usage